lib/pronto/formatter/text_formatter.rb

```diff
@@ -1,10 +1,26 @@
 module Pronto
   module Formatter
     class TextFormatter
+      include Colorizable
+
+      LOCATION_COLOR = :cyan
+
+      LEVEL_COLORS = {
+        info: :yellow,
+        warning: :magenta,
+        error: :red,
+        fatal: :red
+      }.freeze
+
+      attr_reader :config
+
+      def initialize(config = Config.new)
```

Comment:
Yeah, when I run your branch with `TEXT_FORMATTER_COLORS=true` it doesn't colorize the output for me :(

You've tested it and it's supposed too? Maybe I'm just missing something?